src/schema/sources.ts

```diff
@@ -926,6 +937,13 @@ export const resolvers: IResolvers<any, Context> = {
       });
 
       try {
+        if (
+          typeof memberPostingRole !== 'undefined' &&
+          !sourceRoleRankKeys.includes(memberPostingRole)
+        ) {
+          throw new ValidationError('Invalid member posting role');
+        }
```

Comment:
We should uniform this with how we handle the `createSquad` can you maybe abstract it?

Reply:
@rebelchris I thought about moving it to `validateSquadData`, I think it is not an issue if it just throws from there?

Reply:
Ah yes that's totally fine as well.
As long as all fn's use this in a singular way :)